metricbeat/module/php_fpm/process/data.go

```diff
@@ -55,22 +55,40 @@ func eventsMapping(r mb.ReporterV2, content []byte) {
 	}
 	//remapping process details to match the naming format
 	for _, process := range status.Processes {
-		proc := common.MapStr{
-			"pid":                 process.PID,
-			"state":               process.State,
-			"start_time":          process.StartTime,
-			"start_since":         process.StartSince,
-			"requests":            process.Requests,
-			"request_duration":    process.RequestDuration,
-			"request_method":      process.RequestMethod,
-			"request_uri":         process.RequestURI,
-			"content_length":      process.ContentLength,
-			"user":                process.User,
-			"script":              process.Script,
-			"last_request_cpu":    process.LastRequestCPU,
-			"last_request_memory": process.LastRequestMemory,
+		event := mb.Event{
+			RootFields: common.MapStr{
+				"http": common.MapStr{
+					"request": common.MapStr{
+						"method": process.RequestMethod,
+					},
+					"response": common.MapStr{
+						"body": common.MapStr{
+							"bytes": process.ContentLength,
+						},
+					},
+				},
+				"user": common.MapStr{
+					"name": process.User,
+				},
+				"process": common.MapStr{
+					"pid": process.PID,
+				},
+				"url": common.MapStr{
+					"original": process.RequestURI,
+				},
+			},
+			MetricSetFields: common.MapStr{
+				"state":               process.State,
+				"start_time":          process.StartTime,
+				"start_since":         process.StartSince,
+				"requests":            process.Requests,
+				"request_duration":    process.RequestDuration,
```

Comment:
Request duration should go to `event.duration`

Reply:
changed.